genesis.json

```diff
@@ -0,0 +1,7122 @@
+      "starting_proposal_id": "1",
+      "deposits": [],
+      "votes": [],
+      "proposals": [],
+      "deposit_params": {
+        "min_deposit": [
+          {
+            "denom": "usomm",
+            "amount": "10000000"
+          }
+        ],
+        "max_deposit_period": "172800s"
+      },
+      "voting_params": {
+        "voting_period": "172800s"
+      },
+      "tally_params": {
+        "quorum": "0.005000000000000000",
+        "threshold": "0.500000000000000000",
+        "veto_threshold": "0.334000000000000000"
+      }
+    },
+    "gravity": {
+      "params": {
+        "gravity_id": "defaultgravityid",
+        "contract_source_hash": "",
+        "bridge_ethereum_address": "0x0000000000000000000000000000000000000000",
+        "bridge_chain_id": "0",
+        "signed_signer_set_txs_window": "10000",
+        "signed_batches_window": "10000",
+        "ethereum_signatures_window": "10000",
+        "target_eth_tx_timeout": "43200000",
+        "average_block_time": "5000",
+        "average_ethereum_block_time": "15000",
+        "slash_fraction_signer_set_tx": "0.001000000000000000",
+        "slash_fraction_batch": "0.001000000000000000",
+        "slash_fraction_ethereum_signature": "0.001000000000000000",
+        "slash_fraction_conflicting_ethereum_signature": "0.001000000000000000",
+        "unbond_slashing_signer_set_txs_window": "10000"
+      },
+      "last_observed_event_nonce": "0",
+      "outgoing_txs": [],
+      "confirmations": [],
+      "ethereum_event_vote_records": [],
+      "delegate_keys": [],
+      "erc20_to_denoms": [],
+      "unbatched_send_to_ethereum_txs": []
+    },
+    "ibc": {
+      "client_genesis": {
+        "clients": [],
+        "clients_consensus": [],
+        "clients_metadata": [],
+        "params": {
+          "allowed_clients": [
+            "06-solomachine",
+            "07-tendermint"
+          ]
+        },
+        "create_localhost": false,
+        "next_client_sequence": "0"
+      },
+      "connection_genesis": {
+        "connections": [],
+        "client_connection_paths": [],
+        "next_connection_sequence": "0"
+      },
+      "channel_genesis": {
+        "channels": [],
+        "acknowledgements": [],
+        "commitments": [],
+        "receipts": [],
+        "send_sequences": [],
+        "recv_sequences": [],
+        "ack_sequences": [],
+        "next_channel_sequence": "0"
+      }
+    },
+    "mint": {
+      "minter": {
+        "inflation": "0.000000000000000000",
+        "annual_provisions": "0.000000000000000000"
+      },
+      "params": {
+        "mint_denom": "usomm",
+        "inflation_rate_change": "0.00000000000000000",
+        "inflation_max": "0.000000000000000000",
+        "inflation_min": "0.00000000000000000",
+        "goal_bonded": "0.670000000000000000",
+        "blocks_per_year": "6311520"
+      }
+    },
+    "params": null,
+    "slashing": {
+      "params": {
+        "signed_blocks_window": "100",
```

Comment:
These slashing settings say that any more than ~5 minutes consecutive minutes of downtime will produce jailing for a day but no slashing. May I propose something closer to ~50 minutes consecutive downtime producing jailing for a day? Could be convinced to the side of 5 min too.